content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+
+Single cluster deployments offer simplicity, but lack fault isolation and
+failover. If you need higher availability, you should use multiple clusters.
+
+### Multiple clusters
+
+You can configure your mesh to include multiple clusters; such configurations
+are known as **multi-cluster**. Using multiple clusters
+affords the following capabilities beyond that of a single cluster model:
+
+- Fault isolation and failover: `cluster-1` goes down, failover to `cluster-2`.
+- Location-aware routing and failover: Send requests to the nearest service.
+- Configuration isolation: A wrong configuration only affects a single cluster.
+- Various [control plane models](#control-plane-models): Support different
+  levels of availability.
+- Team or project isolation: Each team runs its own set of clusters.
+
+{{< image width="50%"
+    link="./multi-cluster.svg"
+    alt="A service mesh with multiple clusters"
+    title="Multi-cluster"
+    caption="A service mesh with multiple clusters"
+    >}}
+
+Multi-cluster deployments give you a greater degree of isolation and
+availability but increase complexity. If your systems have high availability
+requirements, you likely need clusters across multiple zones and regions. You
+can canary configuration changes or new binary releases in a single cluster,
+where the configuration changes only affect a small amount of user traffic.
+Additionally, if a cluster has a problem, you can temporarily route traffic to
+nearby clusters until you address the issue.
+
+You can configure inter-cluster communication based on the
+[network](#network-models) and the options supported by your cloud provider. For
+example, if two clusters reside on the same underlying network, you can enable
+cross-cluster communication by simply configuring firewall rules.
+
+## Network models
+
+Many production systems require multiple networks or subnets for isolation
+and high availability. Istio supports spanning a service mesh over a variety of
+network topologies.
+
+### Single network
+
+In the simplest case, a service mesh operates over a single fully
+connected network. A single network model satisfies the following conditions:
+
+- The network consists of a number of connected subnets without any
+  overlap in IP or VIP ranges.
+- All {{< gloss >}}workload instance{{< /gloss >}}s can reach each other directly
+  without an Istio gateway.
+
+A single network allows Istio to configure service consumers in a uniform
+way across the mesh with the ability to directly address workload instances.
+
+{{< image width="50%"
+    link="./single-net.svg"
+    alt="A service mesh with a single network"
+    title="Single network"
+    caption="A service mesh with a single network"
+    >}}
+
+### Multiple networks
+
+You can span a single service mesh across multiple networks; such a
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
```

Comment:
Why "an existing network topology" is a capability?

Reply:
Agreed ... I think we can just drop this from the list entirely. It's not really adding any value.

Reply:
@nmittler @vadimeisenbergibm I'm not sure, readers need to know that if they have an existing network topology with multiple networks this is the model for them. WDYT?

Reply:
@rcaballeromx the problem is that an existing network topology may very well be "single network", so it's out of place in either section. Perhaps you could just add a blurb to the introduction to  "network models" indicating that Istio supports various models, allowing them to span a single mesh over an existing topology if they have one.